### ISSUE TITLE
package play.libs.mailer does not exist

### ISSUE THREAD

**Issue by fdo36** [playframework/play-mailer#84]
1.- package play.libs.mailer does not exist
2.- package play.libs.mailer.MailerPlugin does not exist

your whole plugin doesn't work :S is there any alternative?

**Comment by Mogztter** [comment]
@fdo36 Hello Fernando, what version of the plugin are you using ?

> 1.- package play.libs.mailer does not exist
2.- package play.libs.mailer.MailerPlugin does not exist

The package exists, see https://github.com/playframework/play-mailer/blob/master/src/main/java/play/libs/mailer/MailerPlugin.java

**Comment by nem25** [comment]
I have the same problem. I'm using version 3.0.1 for play 2.4.

**Comment by Mogztter** [comment]
@nem25 I received a notification about you saying "same problem here". Would you mind explaining what did you do and how you solved this issue ? thanks

**Comment by nem25** [comment]
It was actually rather silly so hence I removed the message.
I had to reset my Play instance to get it working. Normally, hitting refresh on the browser recognises any changes; I understand not having the updated config settings on refresh but it would then generally fail with a different error like missing config. But here the refresh didn't seem the build the mailer classes. I had to re-make the project and it worked fine.
